src/main/scala/io/opentargets/etl/backend/drug_beta/DrugExtensions.scala

```diff
@@ -110,9 +111,9 @@ object DrugExtensions extends LazyLogging {
                                    extensions: Seq[InputExtension]): Seq[IOResourceConfig] = {
     extensions
       .filter(_.extensionType equalsIgnoreCase extentionType)
-      .flatMap(ext => {
+      .flatMap(ext => { // using optionals so that we don't break the pipeline because of extension files.
         ext.path match {
-          case isJsonPath if isJsonPath.endsWith("json") || isJsonPath.endsWith("jsonl") =>
+          case JsonFile() =>
             Some(IOResourceConfig("json", ext.path))
           case nonJsonPath =>
```

Comment:
@JarrodBaker isn't the compiler telling you this match is not exaustive enough? I mean I would put `case _ =>` for the rest of cases as you just care for json but anyway I will approve it

Reply:
No compiler error at my end. The ext.path variable is a String and the `case nonJsonPath` line catches any String that it hits.